### PR TITLE
feat: account dropdown

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

- Add account drop down

Don't know yet what the data model will look like for this, but right now it takes an array of assets and groups on the `account` key. 

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

related to https://github.com/shapeshift/web/issues/2537 but does not close

## Risk

No risk this is behind a feature flag

## Testing

n/a

### Engineering

You can find this inside the new account page --- flip the multi account flag on and go to the accounts page.

### Operations

* nothing required to be tested at all, this component isn't used anywhere yet.

## Screenshots (if applicable)
Screen Shot 2022-08-30 at 10 38 28 AM

### CODE REVIEW

File: src/pages/Accounts/V2/Accounts.tsx [hunk 1]
```diff
@@ -18,14 +22,54 @@ const AccountHeader = () => {
...
+  {
+    name: 'USD Coin',
+    cryptoBalance: '20',
+    account: '1',
+    symbol: 'USDC',
+  },
+]
+
 export const Accounts = () => {
+  const [activeAccount, setActiveAccount] = useState<string | null>('0')
```
Comment (0xdef1cafe) [inline src/pages/Accounts/V2/Accounts.tsx:41]:
```suggestion
  const [activeAccount, setActiveAccount] = useState<string | null>(null)
```

File: src/components/AccountDropdown/AccountDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  account: string
+  cryptoBalance: string
+  name: string
+  symbol: string
+}
+
+type AccountDropdownProps = {
+  onClick: (arg: string) => void
+  activeAccount: string | null
+  accounts: AccountItem[]
```
Comment (0xdef1cafe) [inline src/components/AccountDropdown/AccountDropdown.tsx:29]: i would strongly prefer if the external interface of this component just accepted `AccountID[]` here

from that, it's able to determine asset ids belong to accounts, assets, and market data from selectors internally

passing a pre-computed object here has two issues

* performance
* every consumer has to collate all of this information (market data, asset data) into this shape to pass to this component, when all it needs is an `AccountId`

Comment (reallybeard) [reply src/components/AccountDropdown/AccountDropdown.tsx:29]: Yah totally, whichever works for y'all when you start hooking it up to data.

**Comment by 0xdef1cafe** [comment]
created separate ticket to capture wrapping this dumb component in a smart one https://github.com/shapeshift/web/issues/2598

**Review by gomesalexandre — APPROVED** [review]
Visually looks good and does what it says on the box, bringing new components over as a story of sorts, hidden under the `MultiAccounts` flag so we can iterate on it and adapt it to our needs.

Agree with @0xdef1cafe re: https://github.com/shapeshift/web/pull/2594/files#r958693197 the API will need to be revamped but that's out of the concern of this PR

**Review by 0xdef1cafe — APPROVED** [review]
approving this PR with the context we are essentially using this as storybook, and new issue https://github.com/shapeshift/web/issues/2598 will capture wiring this up as a smart component with associated interface changes

**Pull request merged**